common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -373,13 +374,31 @@ private static MojoPipeline loadMojoPipelineFromFile() {
       throw new RuntimeException("Could not load mojo from file: " + mojoFile);
     }
     try {
-      MojoPipeline mojoPipeline = MojoPipelineService.loadPipeline(mojoFile);
+      MojoPipeline mojoPipeline = loadMojoPipelineFromFile(mojoFile);
       log.info("Mojo pipeline successfully loaded ({}).", mojoPipeline.getUuid());
       return mojoPipeline;
     } catch (IOException e) {
       throw new RuntimeException("Unable to load mojo from " + mojoFile, e);
     } catch (LicenseException e) {
       throw new RuntimeException("License file not found", e);
+    } catch (Exception e) {
+      throw new RuntimeException("Unexpected exception occurs", e);
```

Comment:
What's the reason for wrapping a generic `Exception` as a `RuntimeException` here?  Would the generic exception not otherwise still propagate up (and be assumed as an unexpected exception)?  Unless the wrapping is adding value (e.g., helpful message), I'd recommended against needing to do so.

Reply:
sounds fair